sde_collections/tasks.py

```diff
@@ -141,3 +143,59 @@ def resolve_title_pattern(title_pattern_id):
     TitlePattern = apps.get_model("sde_collections", "TitlePattern")
     title_pattern = TitlePattern.objects.get(id=title_pattern_id)
     title_pattern.apply()
+
+
+@celery_app.task
+def fetch_and_update_full_text(collection_id, server_type):
+    try:
+        collection = Collection.objects.get(id=collection_id)
+    except Collection.DoesNotExist:
+        raise Exception(f"Collection with ID {collection_id} does not exist.")
```

Comment:
I think we can leave this error handling out. If this exception occurs, it would be because the django admin is seeing a collection which doesn't exist in the database, which would be a fundamental flaw with django, not with the processing pipeline.

Reply:
The corresponding error handling has been removed.